scripts/src/java/org/oppia/android/scripts/proto/coverage.proto

```diff
@@ -33,8 +33,8 @@ message CoveredLine {
 enum Coverage {
   // Coverage status is unspecified.
   UNSPECIFIED = 0;
-  // The line, branch, or function is fully covered, ie. executed atleast once.
+  // The line is fully covered, ie. executed atleast once.
   FULL = 1;
-  // The line, branch, or function is not covered at all.
+  // The line is not covered at all.
```

Comment:
```suggestion
  // The line is not covered at all, that is, it was never executed during a test.
```

Reply:
Thanks, updated the comment as suggested.